src/pages/IstioConfigList/IstioConfigListComponent.tsx

```diff
@@ -6,56 +6,92 @@ import { ActiveFilter, FilterType } from '../../types/NamespaceFilter';
 import * as API from '../../services/Api';
 import Namespace from '../../types/Namespace';
 import { Pagination } from '../../types/Pagination';
-import { RuleItem, RuleList } from '../../types/IstioRuleListComponent';
+import {
+  dicIstioType,
+  filterByName,
+  IstioConfigItem,
+  IstioConfigList,
+  SortField,
+  sortIstioItems,
+  toIstioItems
+} from '../../types/IstioConfigListComponent';
 import PropTypes from 'prop-types';
-import IstioRuleListDescription from './IstioRuleListDescription';
+// import IstioRuleListDescription from './IstioRuleListDescription';
 import { Link } from 'react-router-dom';
 import { PfColors } from '../../components/Pf/PfColors';
 import PfSpinner from '../../components/Pf/PfSpinner';
 
-type SortField = {
-  id: string;
-  title: string;
-  isNumeric: boolean;
-};
-
 const sortFields: SortField[] = [
   {
     id: 'namespace',
     title: 'Namespace',
     isNumeric: false
   },
   {
-    id: 'rulename',
-    title: 'Rule Name',
+    id: 'istiotype',
+    title: 'Istio Type',
+    isNumeric: false
+  },
+  {
+    id: 'istioname',
+    title: 'Istio Name',
     isNumeric: false
   }
 ];
 
-const ruleNameFilter: FilterType = {
-  id: 'rulename',
-  title: 'Rule Name',
-  placeholder: 'Filter by Rule Name',
+const istioNameFilter: FilterType = {
+  id: 'istioname',
+  title: 'Istio Name',
+  placeholder: 'Filter by Istio Name',
   filterType: 'text',
   filterValues: []
 };
 
-type IstioRuleListComponentState = {
+const istioTypeFilter: FilterType = {
+  id: 'istiotype',
+  title: 'Istio Type',
+  placeholder: 'Filter by Istio Type',
+  filterType: 'select',
+  filterValues: [
+    {
+      id: 'RouteRule',
+      title: 'RouteRule'
```

Comment:
I wonder if the labels should be 'RouteRule' or 'Route Rule' (and same for the other types). The latter is more human-friendly.

Reply:
RouteRule or DestinationPolicy, it is how istio uses these names. I'm basically trying to use same terminology to not introduce changes in kiali that may lead some misunderstandings.
(For example, instead to call Mixer Rule, I just called Rule as Istio uses that name).